curator/src/curator_cmd.py

```diff
@@ -95,7 +95,7 @@ def default_index(self):
                     + ' --older-than ' + str(count) \
                     + ' --time-unit ' + unit \
                     + ' --exclude ' + shellquote('^' + re.escape('.searchguard.') + '.*$') \
-                    + ' --exclude ' + shellquote('^' + re.escape('.kibana.') + '.*$')
+                    + ' --exclude ' + shellquote('^' + re.escape('.kibana') + '$')
```

Comment:
actually... there can be any number of kibana indexes depending on the mode so it should really be '.kibana.*'